external/llvm-project/mlir/lib/Dialect/GPU/AmdDeviceLibsIncGen.py

```diff
@@ -40,7 +40,7 @@ def generate(outputPath: Path, rocmPath: Path, libs: List[str]) -> None:
         print(f"{as_signed(byte):+4},", file=out, end=("\n" if i % 8 == 0 else " "))
       # Terminating null pointer needed for
       print("0x00};", file=out)
-    print("static constexpr std::initializer_list<std::pair<llvm::StringRef, llvm::StringRef>> allLibList = {", file=out)
+    print("static const std::initializer_list<std::pair<llvm::StringRef, llvm::StringRef>> allLibList = {", file=out)
```

Comment:
Why?

Reply:
Didn't compile with g++.

Reply:
It's unsupported for rocMLIR builds, and this script doesn't exist upstream